<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 255);
background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 0);
font-size:30pt;</string>
   </property>
   <property name="text">
    <string>Board Page</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20pt;
color:rgb(170, 255, 0)</string>
   </property>
   <property name="text">
    <string>Your Board List:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="boardlist">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>461</width>
     <height>241</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;jokl&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font-size:15pt;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="lineWidth">
    <number>18</number>
   </property>
   <property name="midLineWidth">
    <number>13</number>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::InternalMove</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addboardbutton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>660</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font-size:17pt;
background-color:rgb(85, 170, 255)</string>
   </property>
   <property name="text">
    <string>Create Board</string>
   </property>
  </widget>
  <widget class="QPushButton" name="accountbutton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
font-size:15pt;
background-color:rgb(255, 255, 0)</string>
   </property>
   <property name="text">
    <string>Account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteboardbutton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>660</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
font-size: 15pt;
background-color:rgb(191, 191, 191);</string>
   </property>
   <property name="text">
    <string>Delete Board</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20pt;
color:rgb(170, 255, 0)</string>
   </property>
   <property name="text">
    <string>Public Board List:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="boardlist_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>461</width>
     <height>241</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;jokl&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font-size:15pt;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="lineWidth">
    <number>18</number>
   </property>
   <property name="midLineWidth">
    <number>13</number>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::InternalMove</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
